src/main/java/com/google/sps/data/Route.java

```diff
@@ -21,26 +21,32 @@
   private long routeId;
   private String routeName;
   private boolean isPublic;
+  private boolean isCompleted;
   private long startHour;
   private long startMinute;
   private List<Marker> routeMarkers;
   private List<Long> editorsArray;
+  private double rating;
 
   public Route(
       long routeId,
       String routeName,
       boolean isPublic,
+      boolean isCompleted,
       long startHour,
       long startMinute,
       List<Marker> routeMarkers,
-      List<Long> editorsArray) {
+      List<Long> editorsArray,
+      double rating) {
```

Comment:
What is the plan for editing the rating? I assume it will be calculated from multiple other ratings, rather than being edited directly - is that correct?

Reply:
Oh, yeah, i think in that case it maybe better to create two variables (and delete existing one): number of ratings and their sum.